app/src/main/java/com/connorlay/knitgrid/activities/PatternGridViewActivity.java

```diff
@@ -21,5 +26,17 @@ protected void onCreate(Bundle savedInstanceState) {
         transaction.replace(R.id.activity_pattern_grid_view_content_frame_layout, new PatternGridFragment());
         transaction.addToBackStack(null);
         transaction.commit();
+
+        if (isFirstLaunch()) {
```

Comment:
I was thinking not an activity but just an object with the responsibility of setting up the initial state of the application. If we extract it out then we can easily swap out other implementations of our object. That and it might be easier to unit test? I don't feel super strongly either way so it's your call.